lib/push-manager.js

```diff
@@ -136,13 +138,19 @@ PushManager.prototype.configureProvider = function (deviceType, pushSettings) {
 /**
  * Lookup or set up push notification service for the given appId
  * @param {String} appId The application id
- * @returns {*}
+ * @param {String} deviceType The type of device (android, ios)
+ * @param {PushManager~configureApplicationCallback} cb The callback function
  */
 PushManager.prototype.configureApplication = function (appId, deviceType, cb) {
   assert.ok(cb, 'callback should be defined');
   var self = this;
   var msg;
 
+  /**
+   * @callback PushManager~configureApplicationCallback
+   * @param {Error|null} err The error object
+   * @param {Application} application The Application matched to the the appId
+   */
```

Comment:
We use a different format for describing callback arguments, see [here](https://github.com/strongloop/loopback/blob/553a3b208386a0e946cccfcfdba1d83eae8a6f15/lib/persisted-model.js#L85-L97). I know our format is non-standard, but it is required for http://apidocs.strongloop.com/

Could you please remove this comment and describe the callback argument in the method doc block instead?